src/wallet/spend.h

```diff
@@ -213,6 +213,7 @@ struct CreatedTransactionResult
         : tx(_tx), fee(_fee), fee_calc(_fee_calc), change_pos(_change_pos) {}
 };
 
+void DiscourageFeeSniping(CMutableTransaction& tx, FastRandomContext& rng_fast, interfaces::Chain& chain, const uint256& block_hash, int block_height);
```

Comment:
nit: blank line under this; ideally docs above

Reply:
Done